### PR TITLE
Update to JuMP v0.23

### PR DESCRIPTION
**Pull request by jd-lara**
Update to MOI and MA 1.0

### CODE REVIEW

**Comment by jd-lara** [comment]
@odow The tests are passing now in this PR to update PJ.

File: Project.toml [hunk 1]
```diff
@@ -10,9 +10,9 @@ MutableArithmetics = "d8a4904e-b15c-11e9-3269-09a3773c0cb0"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
-JuMP = "0.22"
-MathOptInterface = "0.10"
-MutableArithmetics = "0.3"
+JuMP = "0.23"
+MathOptInterface = "1"
+MutableArithmetics = "1"
 julia = "1"
```
Comment (odow) [inline Project.toml]:
```suggestion
julia = "1.6"
```

File: Project.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name = "ParameterJuMP"
 uuid = "774612a8-9878-5177-865a-ca53ae2495f9"
 repo = "https://github.com/JuliaStochOpt/ParameterJuMP.jl.git"
-version = "0.3.2"
+version = "0.3.3"
```
Comment (odow) [inline Project.toml]:
```suggestion
version = "0.4.0"
```

**Comment by jd-lara** [comment]
bump to @odow to merge and tag. I need this to update PSI to JuMP 0.23 and MOI 1.0

**Pull request merged**